docs/settings.ini

```diff
@@ -14,7 +14,7 @@ r_docs = 0
 scala_docs = 1
 
 [document_sets_v1.2.0]
```

Comment:
I see two copies of document_sets_1.2.0.

Reply:
Aaron explained offline that for each version one entry is for release branch while the other is for release tag. Currently, the pipeline is set up so that docs are built from branches.